pyproject.toml

```diff
@@ -44,10 +44,13 @@ google-cloud-aiplatform = {version = "^1.66.0", optional = true }
 cohere = {version = "^5.9.0", optional = true}
 mistralai = {version = "^1.0.3", optional = true}
 qdrant-client = {version = "^1.11.3", optional = true}
-llama-index = {version = "^0.10.55", optional = true }
```

Comment:
I think we need this dep because an example shows how to use a text splitter from llama-index?

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/226/commits/c1ef1d637c63c5fa6c14c0a25c8bd9562705534b